docker-compose.yml

```diff
@@ -24,28 +24,29 @@ services:
       - ../ocis-reva:/ocis-reva
       - ../reva:/reva
     environment:
-      EOS_MGM_URL: ${EOS_MGM_URL:-root://mgm-master.testnet:1094}
+      # ocis log level will be used for all services
+      OCIS_LOG_LEVEL: debug
+      # domain setup
+      # TODO currently the below lines hardcode the port to 9200, use an OCIS_URL that includes protocol and port
       OCIS_DOMAIN: ${OCIS_DOMAIN:-localhost}
+      PROXY_OIDC_ISSUER: https://${OCIS_DOMAIN:-localhost}:9200
       KONNECTD_ISS: https://${OCIS_DOMAIN:-localhost}:9200
-      KONNECTD_LOG_LEVEL: debug
-      KONNECTD_TLS: '0'
```

Comment:
the default is already `false`as is necessary because TLS handling is offloaded to the proxy

Reply:
I'd be a fan of leaving all the defaults. Or document the defaults as a comment.